docworks-cli/test/dts-e2e.spec.js

```diff
@@ -64,7 +65,23 @@ describe('dts workflow e2e', function () {
         */
       declare module 'Service' {
           function operation(param: string): void;
-      
+
       }`))
   })
+
+  it.only('generate dts from a with ignore some services', async function () {
+    logger.log('run test')
+    logger.log('--------')
+
+    await runCommand('./bin/docworks dts -l ./test/docworks-service -o ./tmp/globals3'.split(' '))
+    await runCommand('./bin/docworks dts -i Service2 -l ./test/docworks-service -o ./tmp/globals4'.split(' '))
+
+    const fullContent = await fsExtra.readFile('tmp/globals3.d.ts', 'utf-8')
+    const filteredContent = await fsExtra.readFile('tmp/globals4.d.ts', 'utf-8')
+
+    const moduleRegex = /declare module '.*?'/gm
+    const allModules = fullContent.match(moduleRegex)
+    const filterModules = filteredContent.match(moduleRegex)
+    expect(filterModules.length).to.equal(allModules.length - 1)
```

Comment:
I think it's worth to check that we also filter the right module, and not just the length

Reply:
Done